ecommerce-demo/workspaces/demo/workspaceAnalytics.json

```diff
@@ -8634,11 +8634,11 @@
           "titleBase": "",
           "titlePattern": "%titleBase - %granularityTitle"
         },
-        "id": "imventory_month",
+        "id": "inventory_month",
         "tags": [
-          "Imventory month"
+          "Inventory month"
         ],
-        "title": "Imventory month"
+        "title": "Inventory month"
```

Comment:
Should we change `"Monthlyinventory"` → `"Monthly inventory"` at the following places as well, for the sake of consistency?

- https://github.com/gooddata/gooddata-public-demos/blob/5554bda5aecb596bb9685cb421b2a733358cf1ff/ecommerce-demo/workspaces/demo/workspaceAnalytics.json#L7983

- https://github.com/gooddata/gooddata-public-demos/blob/5554bda5aecb596bb9685cb421b2a733358cf1ff/ecommerce-demo/workspaces/demo/workspaceAnalytics.json#L8057

- https://github.com/gooddata/gooddata-public-demos/blob/5554bda5aecb596bb9685cb421b2a733358cf1ff/ecommerce-demo/workspaces/demo/workspaceAnalytics_aio.json#L7498

- https://github.com/gooddata/gooddata-public-demos/blob/5554bda5aecb596bb9685cb421b2a733358cf1ff/ecommerce-demo/workspaces/demo/workspaceAnalytics_aio.json#L7561